pages/common/gource.md

```diff
@@ -0,0 +1,32 @@
+# gource
+
+> Renders an animated tree diagram of Git, SVN, Mercurial and Bazaar repositories.
+> It shows files and folders being created, modified or removed over time.
+
+- Run gource in a directory:
+
+`gource {{/path/to/repository/}}`
+
+- Run gource in fullscreen mode (assuming the command is run from the project's root directory):
+
+`gource -f`
```

Comment:
Maybe split this into a plain `gource` invocation (placed above all other commands), with the remark in parenthesis, and a `gource -f` combined with the `gource -b` below.